test/test-greenlight.js

```diff
@@ -1,16 +1,8 @@
-/* global describe it expect before beforeEach afterEach after */
+/* global describe it before beforeEach afterEach after */
```

Comment:
I think this can be solved by installing the types

`npm i -D @types/mocha @type/chai`